core/dblookupext/eventsHashesByTxHashIndex_test.go

```diff
@@ -98,15 +89,76 @@ func TestSaveAndGetEventsReceiptsHashesByTxHash(t *testing.T) {
 		"wrongTx": &transaction.Transaction{},
 	}
 
-	err := eventsHashesIndex.saveEventsHashes(epoch, nil, receipts)
+	err := eventsHashesIndex.saveResultsHashes(epoch, nil, receipts)
 	require.Nil(t, err)
 
-	expectedEvents := &EventsHashesByTxHash{
-		ReceiptsHash:   recHash1,
-		ScrHashesEpoch: nil,
+	expectedEvents := &ResultsHashesByTxHash{
+		ReceiptsHash:            recHash1,
+		ScResultsHashesAndEpoch: nil,
 	}
 
 	eventsHashes, err := eventsHashesIndex.getEventsHashesByTxHash(txWithReceiptHash, epoch)
 	require.Nil(t, err)
 	require.Equal(t, expectedEvents, eventsHashes)
 }
+
+func TestGroupSmartContractResults(t *testing.T) {
```

Comment:
Before I was referring to directly testing the grouping function, but all right.